docs/deploy/cluster.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 1
+description: "Learn how to deploy a distributed Restate cluster."
+---
+
+import Admonition from '@theme/Admonition';
+
+# Cluster deployment
+
+This page describes how you can deploy a distributed Restate cluster.
+
+<Admonition type="note" title="Preview state">
+    The current distributed deployment state is a preview.
+    We reserve the right to make breaking changes.
+</Admonition>
+
+## Configuration
+
+To deploy a distributed Restate cluster without external dependencies, you need to configure the following settings in your [server configuration](/operate/configuration/server):
+
+```toml
+# Let every node run all roles
+roles = ["metadata-server", "admin", "worker", "log-server"]
+# Every node needs to have a unique node name
+node-name = <UNIQUE_NODE_NAME>
+# All nodes need to have the same cluster name
+cluster-name = <CLUSTER_NAME>
+# Make sure it does not conflict with the other nodes
+advertised-address = <ADVERTISED_ADDRESS>
+# At most one node can be configured with auto-provision = true
+auto-provision = false
+
+[bifrost]
+# Only the replicated Bifrost provider can be used in a distributed deployment
+default-provider = "replicated"
+
+[bifrost.replicated-loglet]
+# Replicate the data to 2 nodes. This requires that the cluster has at least 2 nodes to
+# become operational. If the cluster has at least 3 nodes, then it can tolerate 1 node failure.
+default-replication-property = 2
+
+[metadata-server]
+# To tolerate node failures, use the embedded metadata server
+type = "embedded"
+
+[metadata-store-client]
+# Use the embedded metadata store
+type = "embedded"
+# List all the advertised addresses of the nodes that run the metadata-server role
+addresses = [<ADVERTISED_ADDRESS>, ..., <ADVERTISED_ADDRESS_NODE_X>, ...]
```

Comment:
Do you know why the red highlighted test is printed in italics when rendering it @gvdongen?

Reply:
You could try switch `<` and `>` for `&lt;` and `&gt;` html escape codes if there isn't a more elegant solution?

Reply:
Will try it out. Thanks for the tip.